.circleci/config.yml

```diff
@@ -78,17 +78,13 @@ jobs:
           command: |
             pip install -c constraints.txt codecov
       - run:
-          name: Install requirements
+          name: Install requirements and package
           command: |
-            pip install -c constraints.txt -r requirements.txt
+            make install
```

Comment:
The circleci caching tries to cache the installation of the requirements. Please only install the requirements in this step (ie. call make install-requirements and install the rest later).